packages/core/src/shared/telemetry/vscodeTelemetry.json

```diff
@@ -717,6 +747,44 @@
                 },
                 {
                     "type": "cwsprChatConversationType"
+                },
+                {
+                    "type": "cwsprChatHasProjectContext",
+                    "required": false
+                },
+                {
+                    "type": "cwsprChatProjectContextQueryMs",
```

Comment:
can the `duration` field be used instead?

Reply:
No. Duration stands for the entire event duration while this new field only applies to the LSP query API latency